h/static/scripts/directive/excerpt.coffee

```diff
@@ -0,0 +1,14 @@
+module.exports = [ '$timeout', ($timeout) ->
+    link: (scope, elem, attr, ctrl) ->
+      $timeout -> if elem[0].scrollHeight > elem[0].clientHeight
+        elem.prepend angular.element '<span class="more-quote"> More...</span>'
+        elem.append angular.element '<span class="less-quote"> Less ^</span>'
+        elem.find('.more-quote').on 'click', ->
+          $(this).hide()
+          elem.addClass('show-full-quote')
+        elem.find('.less-quote').on 'click', ->
```

Comment:
I would make these more semantically general and meaningful or the utility of `excerpt` is greatly diminished. It may be that `excerpt` is the wrong name if it's implying that we're showing a quotation when in fact all we're doing is truncating / collapsing / or by whatever other term affecting the visual presentation of any arbitrary thing (quotation, comment, image, whatever) in its original context (not excerpted).